### DISCUSSION TITLE
Request for a general Craft user guide from a client perspective

### DISCUSSION THREAD

**Discussion by darylknight** [craftcms/cms#9564] [Ideas]
I've had several requests recently for a "Craft user guide" specifically to show prospective clients why they should choose Craft, from a website owner perspective rather than as a developer. The latest article with 3.7 that demonstrates the new workflow for entries is excellent: https://craftcms.com/knowledge-base/editing-entries, but as far as I know, it's the only thing like it.

Is there anything, anywhere by P&T that shows off the features of Craft from an editor's perspective? 

I embed the guide I wrote into all client control panels using the Control Panel nav plugin: https://www.notion.so/Craft-User-Guide-26a6dbe6ab174aa996cccb2762b2468b, but I'd love to see an official reference.

I guess the reason this doesn't exist is because every Craft install is different, bespoke to clients, so it wouldn't cover things like "to edit pages", it should be general like, "this is what you can do with Assets" or "This is how the editing features work for content editors".

**Comment by samuel-clara** [comment]
This would still be very useful. From my point of view this is something that P&T should provide for each major version of the CMS. Of course it can't cover each detail of a customizable backend like Craft but it should at least introduce the platform to an author.  And in my opinion it should not only be in text format but also in video format.

This would also help us enormously in the sale process of craft cms a lot. At the moment the craft cms site is also decidedly developer-centered which is understandable but there is a total lack of content aimed at authors.
